packages/core/database/lib/dialects/mysql/index.js

```diff
@@ -32,6 +32,14 @@ class MysqlDialect extends Dialect {
     };
   }
 
+  async initialize() {
+    try {
+      await this.db.connection.raw(`set session sql_require_primary_key = 0;`);
+    } catch (err) {
+      console.error({ err });
+    }
```

Comment:
The idea here would be to ignore and do an empty catch so we still get the same error later but don't do logs for users that shouldn't see them.